src/toil/test/wdl/wdltoil_test.py

```diff
@@ -90,6 +114,17 @@ def check(self, p: subprocess.CompletedProcess) -> None:
 
         p.check_returncode()
 
+    @slow
+    def test_unit_tests_v11(self):
+        # There are still some bugs with the WDL spec, use a fixed version until
+        repo_url = "https://github.com/stxue1/wdl.git"
+        repo_branch = "wdl-1.1.3-fixes"
+        command = f"{exactPython} setup_unit_tests.py -v 1.1 --extra-patch-data unit_tests_patch_data.yaml --repo {repo_url} --branch {repo_branch} --force-pull"
+        p = subprocess.run(command.split(" "), capture_output=True)
```

Comment:
`check=True` isn't set on this call, so it is allowed to fail. And `capture_output=True` is set but the output is never used, nor will it go to the same place as our output. So I think if this *does* fail it will fail silently.

This probably should be a `check_call`, or we should have a comment justifying why we want to completely ignore the failure of this command and any error messages it may produce.

Or maybe we're just missing another call to `self.check(p)`?

Reply:
Yeah, this was missing a check